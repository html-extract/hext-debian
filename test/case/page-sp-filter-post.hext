<a href:filter(/post-(\d+)/):post_id />
