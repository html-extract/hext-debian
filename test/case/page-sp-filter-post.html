<a href="/post-123/post-title">post</a>
<a href="/post-234/post-title">post</a>
<a href="/post-345/post-title">post</a>
<a href="/post-4567/post-title">post</a>
