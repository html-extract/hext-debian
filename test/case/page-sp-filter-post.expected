{"post_id":"123"}
{"post_id":"234"}
{"post_id":"345"}
{"post_id":"4567"}
